mutiny-wasm/src/lib.rs

```diff
@@ -1200,6 +1201,25 @@ impl MutinyWallet {
             .into())
     }
 
+    /// Approves a nostr wallet auth request.
+    /// Creates a new NWC profile and saves to storage.
+    /// This will also broadcast the info event to the relay.
+    pub async fn approve_nostr_wallet_auth(
+        &self,
+        name: String,
+        uri: String,
+    ) -> Result<NwcProfile, MutinyJsError> {
+        let uri = NIP49URI::from_str(&uri).map_err(|_| MutinyJsError::InvalidArgumentsError)?;
+        log::info!("Approving NWC auth request: {}", uri);
+        let profile = self
+            .inner
+            .nostr
+            .approve_nostr_wallet_auth(ProfileType::Normal { name }, uri, NwcProfileTag::General)
```

Comment:
might as well remove type from this request if it's only going to be used for `Normal`

Reply:
We may want to use `approve_nostr_wallet_auth` with other profile types in the future, think it is better to have the type.